pkg/services/store/entity/sqlstash/create.go

```diff
@@ -0,0 +1,126 @@
+package sqlstash
```

Comment:
Tests in another PR, too long already

Reply:
It would be nice to have at least one test for each method : create / update / delete.